astraguard/swarm/safety_simulator.py

```
@@ -0,0 +1,515 @@
+        if not self.registry:
+            return []
+
+        # Simplified: return all other agents
+        peers = self.registry.get_alive_peers()
+        neighbors = [
+            p.satellite_serial for p in peers
+            if p.satellite_serial != agent_serial
+        ]
+        return neighbors
+
+    def _update_latency_metrics(self, latency_ms: float) -> None:
```

Ruff (0.14.10)

Unused method argument: `latency_ms`

(ARG002)